### DISCUSSION TITLE
Start a job on 10 a.m. on the last-but-third day every month?

### DISCUSSION THREAD

**Discussion by KarloX2** [quartznet/quartznet#1017] [Q&A] [answered]
Hi Community!

can somebody please provide some lines of C# code that demonstrate how I can setup a schedule that would start a job on the "last-but-third" day in every month at e.g. 10:00 am? E.g. in November that would be 27th of Nov., in Decmeber it would be on the 28th of Dec.

And BTW: What would happen in November when I configure to start a job on the 31st ?

Thanks!

**Comment by philsayersCDS** [comment, marked as answer]
Assuming the job you want to run on the "3 days before end of month" is named something like "MyAlmostEndOfMonthJob":

Have a second job that is triggered on the first day of every month called "MyAlmostEndOfMonthScheduleBuilderJob".
Put code in that scheduler builder to handle the date math and create a trigger for "MyAlmostEndOfMonthJob".

The date math would be something like..

* Find last day of current month.
* Subtract 3 days.
* Schedule MyAlmostEndOfMonthJob to run at whatever time of day you want on that calculated date.
